docs/production-deployment/cloud/nexus/getting-started.mdx

```diff
@@ -0,0 +1,66 @@
+---
+id: getting-started
+slug: /cloud/nexus/getting-started
+title: Getting Started with Temporal Nexus
+description: Get started with Temporal Nexus, including setting up Nexus Endpoints and integrating Nexus into your Temporal Workflows.
+sidebar_label: Getting Started
+tags:
+  - temporal cloud
+  - nexus setup
+keywords:
+  - Temporal Nexus onboarding
+  - setting up Nexus Endpoints
+  - temporal cloud
+  - nexus setup
+
+---
+
+Temporal Nexus Public Preview is available within your Temporal Cloud account, with support in the [Temporal Go SDK](/develop/go).
+
+Calls across existing Namespaces can be enabled by creating a Nexus Endpoint in the Nexus API Registry, creating a Nexus Service in a Worker in the handler Namespace, and then using the Nexus Service from a caller Workflow in a different Namespace.
+
+Monolithic Namespaces can be decomposed into multiple Namespaces, by hiding service implementations behind a Nexus Endpoint in the monolithic Namespace, pointing all consumers at the new Nexus Endpoint, and then changing the Endpoint’s target Namespace to a different Namespace. Multiple Nexus Endpoints can target a single monolithic Namespace.
+
+## Cross-Namespace
+
+Calls across existing Namespaces can be enabled by creating a Nexus Endpoint in the Nexus API Registry, creating a Nexus Service in a Worker in the handler Namespace, and then using the Nexus Service from a caller Workflow in a different Namespace.
+
+Steps to enable calls across existing Namespaces:
+
+1. Add Nexus Services to the same Workers as the Temporal primitives being abstracted. 
+2. Add a Nexus Endpoint that:  
+   1. Targets the handler Namespace. 
+   2. Allows the caller Namespace.  
```

Comment:
Again, guessing on this

```suggestion
   2. Lists the Namespaces ("callers") that may place requests to the handler Namespaces.
```

Reply:
not sure if this is correct so resolving it.